semantic-ui/index.d.ts

```diff
@@ -0,0 +1,5434 @@
+// Type definitions for Semantic UI 2.2
+// Project: http://semantic-ui.com/
+// Definitions by: Leonard Thieu <https://github.com/leonard-thieu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery" />
+
+// tslint:disable:unified-signatures
+
+interface JQuery {
+    // TODO: Do all modules have a 'setting' behavior?
```

Comment:
It's not explicitly documented but [tests](https://github.com/Semantic-Org/Semantic-UI/blob/2.2.7/test/modules/module.spec.js) seem to indicate there should be a 'setting' behavior available to all modules. However, I couldn't get get the tests working to verify that.